content/en/docs/v3.4/get-started/etcd.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Get Started with etcd
+weight: 910
+description: Getting started with etcd on Linux or macOS
+---
+
+# Quickstart
```

Comment:
By convention in the docs pages, the page title serves as a level 1 heading 🤷‍♂️. I'd suggest dropping this heading.